docs/theory/force_calibration/force_calibration.rst

```diff
@@ -0,0 +1,363 @@
+Neglecting hydrodynamic and inertial effects (more on this later), we obtain the following equation of motion:
+
+.. math::
+
+    \dot{x} = \sqrt{2D} \xi (t)
+
+Here :math:`x` represents the time-dependent position of the bead, :math:`\dot{x}` is the first derivative with respect to time, :math:`D` is the diffusion constant and :math:`\xi(t)` is normalized white noise.
+The expected power spectrum of such free diffusion is given by the following equation:
+
+.. math::
+
+    P_\mathrm{diffusion}(f) = \frac{D}{\pi^2 f^2}
+
+We can plot this spectrum for different diffusion constants::
+
+    f = np.arange(100, 23000)
+    for diffusion in 10**np.arange(1, 4):
+        plt.loglog(f, diffusion / (np.pi * f**2), label=f"D={diffusion} $\mu m^2/s$")
+
+    plt.xlabel("Frequency [Hz]")
+    plt.ylabel("Amplitude [$\mu m^2$/s]");
+    plt.legend()
+
+.. image:: figures/diffusion_spectra.png
+
+Here, the vertical axis represents a displacement amplitude while the horizontal axis represents frequency.
+Observe how, for free diffusion, the low frequencies have larger amplitudes than the high frequencies.
+
+Next, consider the effect of the optical trap on this diffusion.
+The optical trap will pull the bead back towards the focus of the beam.
+As such, the trap will constrain motion, limiting in particular the high amplitudes.
+In other words, we expect this effect to be more prominent at low frequencies as they have larger amplitudes.
+Compared to the previous spectra, this should result in a plateau at the low frequency end of the spectrum.
+Still neglecting hydrodynamic and inertial effects, one can write down the differential equation for a trapped bead.
+
+.. math::
+
+    \dot{x} + \frac{\kappa}{\gamma} x = \sqrt{2D} \xi (t)
+
+Note that we now have a few additional parameters, namely the drag coefficient :math:`\gamma` and the trap stiffness :math:`\kappa`.
+From this, the following power spectrum can be derived:
+
+.. math::
+
+    P_{\mathrm{diffusion}}(f) = \frac{D}{\pi^2 \left(f^2 + \left(\frac{\kappa}{2 \pi \gamma}\right)^2\right)} = \frac{D}{\pi^2 \left(f^2 + f_c^2\right) }
+
+Note how we've defined a corner frequency :math:`f_c` from the trap stiffness and the drag coefficient.
+When plotting this equation for various values of the corner frequency, we see the expected plateau::
+
+    plt.figure(figsize=(12, 4))
+    plt.subplot(1, 2, 1)
+    diffusion, corner_freq = 1000, 1000
+    for diffusion in 10**np.arange(1, 4):
+        plt.loglog(f, diffusion / (np.pi * (f**2 + corner_freq**2)), label=f"D={diffusion} $\mu m^2/s$")
+
+    plt.xlabel("Frequency [Hz]")
+    plt.ylabel("Amplitude [$\mu m^2$/s]");
+    plt.legend()
+
+    plt.subplot(1, 2, 2)
+    diffusion, corner_freq = 1000, 1000
+    for corner_freq in [1000, 5000, 10000]:
+        line, = plt.loglog(
+            f, diffusion / (np.pi * (f**2 + corner_freq**2)), label=f"corner_freq={corner_freq} Hz"
```

Comment:
I think here for the label we should either use "corner frequency" or `$f_c$` for consistency